content/en/about/contribute/creating-and-editing-pages/index.md

```diff
@@ -694,7 +694,7 @@ There are two types of promotions supported:
 Stickers have minimal visuals displayed to the user during the event period.
```

Comment:
I believe the original PR had changes in the countdown stickers as well to indicate under which circumstances countdown stickers should be used.

Reply:
I admit that they did, but was yet in another PR that updated a lot of the contributing doc. I can pull that small snippet as well.

Reply:
Fixed.